pep-0698.rst

```diff
@@ -447,8 +447,9 @@ We decided against it because:
   be considerable benefits.
 - We believe that it would be rarely used and catch relatively few bugs.
 
-  - The author of the ``overrides`` package
-    `has noted <https://discuss.python.org/t/pep-698-a-typing-override-decorator/20839/4>`_
+  - The author of the
+    `Overrides package <https://pypi.org/project/overrides/>`_ has
+    `noted <https://discuss.python.org/t/pep-698-a-typing-override-decorator/20839/4>`_
```

Comment:
```suggestion
    `noted <https://discuss.python.org/t/pep-698-a-typing-override-decorator/20839/4>`__
```

This would be a spot where an anonymous hyperlink would be appropriate, as `noted` is an ambiguous and easy to re-use link target name.